### DISCUSSION TITLE
Error 1. Retrying in five minutes.

### DISCUSSION THREAD

**Discussion by candjfields** [Chia-Network/chia-blockchain#5750] [Support]
At the 69% progress mark of one plot, I get the following message (not exact): 

"Only wrote 1472 of 1048572 bytes at offset 385874496 to X:\xxxxxxxxxxxxxx.tmp file name with length 385875968. Error 1. Retrying in five minutes."

It just stays there and just continues for however long you let it. After waiting hours, again receiving the same error message, I just end up restarting Chia. After the restart, when I check for the plot, it's missing. 

See screen shot attached:
![Chia_QoJcKGp49s](https://user-images.githubusercontent.com/84602414/119201911-a133c880-ba44-11eb-8984-3ff2d97f1497.png)

What is odd, is that this same X:\ drive (USB 3.2 connected Sabrent 1 TB NVME SSD), which is my temp drive for Chia, randomly disconnected and then reconnected before this happened.

Chia: 1.1.6
OS: Windows 10 Enterprise 20H2
RAM: 16GB DDR4
CPU: Intel Core i5 4430 (4th Gen) 
USB Add-in card: Inateck PCIe 3.0 to USB 3.2 Gen 2 Card with 20 Gbps Bandwidth, 3 USB Type-A and 2 USB Type-C Ports

**Comment by tschneringer0000** [comment]
Check your temp storage. It’s full.

**Comment by BrandtH22** [comment]
@candjfields , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
